src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -2041,6 +2060,34 @@ private function skip_whitespace() {
 	 * @since 6.2.0
 	 */
 	private function after_tag() {
+		/*
+		 * There could be lexical updates enqueued for an attribute that
+		 * also exists on the next tag. In order to avoid conflating the
+		 * attributes across the two tags, lexical updates with names
+		 * need to be flushed to raw lexical updates.
+		 */
+		$this->class_name_updates_to_attributes_updates();
+		if ( 1000 < count( $this->lexical_updates ) ) {
```

Comment:
Shouldn't the constant be re-used here?

```suggestion
		if ( self::MAX_SEEK_OPS < count( $this->lexical_updates ) ) {
```

Or maybe this constant isn't for the same thing. In any case, why 1000 here?

Reply:
Thanks for asking about it. This is not the seek limit. I've added an explanation for this in 0ec643683d, which I meant to do originally but forgot because I had stopped working abruptly.